pulsar-client-shaded/pom.xml

```diff
@@ -168,6 +168,7 @@
                   <include>org.tukaani:xz</include>
                   <!-- Issue #6834, Since Netty ByteBuf shaded, we need also shade this module -->
                   <include>org.apache.pulsar:pulsar-client-messagecrypto-bc</include>
+                  <include>it.unimi.dsi:*</include>
```

Comment:
This isn't the correct solution for shading fastutil. Please see https://github.com/apache/pulsar/pull/23600/files#r1848136065